waspc/src/Wasp/Generator/DbGenerator/Operations.hs

```diff
@@ -30,22 +30,11 @@ import Wasp.Generator.DbGenerator.Common
     getOnLastDbConcurrenceChecksumFileRefreshAction,
   )
 import qualified Wasp.Generator.DbGenerator.Jobs as DbJobs
-import Wasp.Generator.FileDraft.WriteableMonad
-  ( WriteableMonad (copyDirectoryRecursive),
-  )
-import Wasp.Generator.Job (JobMessage)
-import qualified Wasp.Generator.Job as J
-import Wasp.Generator.Job.IO (printJobMessage, readJobMessagesAndPrintThemPrefixed)
+import Wasp.Generator.FileDraft.WriteableMonad (WriteableMonad (copyDirectoryRecursive))
+import Wasp.Generator.Job.IO (printJobMsgsUntilExitReceived, readJobMessagesAndPrintThemPrefixed)
 import qualified Wasp.Generator.WriteFileDrafts as Generator.WriteFileDrafts
 import Wasp.Util (checksumFromFilePath, hexToString)
 
-printJobMsgsUntilExitReceived :: Chan JobMessage -> IO ()
```

Comment:
Moved into `Job/IO.hs` since it felt pretty general (and similar to another helper function in there already) - plus, I needed it, too. :)

Reply:
Sure, makes sense!